x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/fields_browser.cy.ts

```diff
@@ -119,12 +106,11 @@ describe.skip('Fields Browser', { tags: ['@ess', '@serverless'] }, () => {
     });
   });
 
-  context('Editing the timeline', () => {
+  describe('Editing the timeline', () => {
     beforeEach(() => {
       login();
       visitWithTimeRange(hostsUrl('allHosts'));
       openTimelineUsingToggle();
-      populateTimeline();
       openTimelineFieldsBrowser();
     });
 
```

Comment:
same comment here, I feel like the first 3 tests could easily be merged into a single one that would test add, remove and reset fields.

Thoughts?

Reply:
actually even the fourth one (`restores focus to the Customize Columns button when `Reset Fields` is clicked`) can easily be merged with the other 3. Just check the focus after you've reset. It would just be a this single line added at the end
```
cy.get(TIMELINE_FIELDS_BUTTON).should('have.focus');
```

It doesn't really make sense to have a whole test that verifies just that